week2/clockapp.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <body>
+    <p>How logn do you want the timer to last?(seconds)</p>
+    <input id = 'timertime'>
+    <button id = 'starttimerbutt'>Start Timer</button>
+
+    <script>
+    $('#starttimerbutt').on('click', ()=>{
+      $timertime = $('#timertime').val();
```

Comment:
Since this is actually a value and not a jQuery object you don't need the `$` in front of the variable.

```js
// valid
const $timerTime = $("#timertime");
```

```js
// $ was not necessary since you are grabbing the value. I would also make variables more descriptive.
const timerTimeValue = $("#timertime").val();
```